src/Console/ConsoleOptionParser.php

```diff
@@ -576,6 +576,7 @@ public function addSubcommand($name, array $options = [])
             $command = $name;
             $name = $command->name();
         } else {
+            $name = Inflector::underscore($name);
```

Comment:
I would consider this definitely a misconfig, having both my_method and myMethod as methods + console option config seems nonsense to me.
You either use the underscore or camelCase - and ideally consistently :)

This was part of the reason I think it is better to communicate this in the 3.5 migration guide.